sdk/appcenter/src/main/java/com/microsoft/appcenter/AppCenter.java

```diff
@@ -53,6 +53,18 @@
      */
     public static final String LOG_TAG = AppCenterLog.LOG_TAG;
 
+    /**
+     * Default maximum storage size for SQLite database.
+     */
+    @VisibleForTesting
+    static final long DEFAULT_MAX_STORAGE_SIZE_IN_BYTES = 10 * 1024 * 1024;
+
+    /**
+     * Minimum size allowed for set maximum size.
+     */
+    @VisibleForTesting
+    static final long MINIMUM_STORAGE_SIZE = 16385;
```

Comment:
Where does this number come from? It's *slightly* over 4 pages. So I suggest either 4 or 5 pages should be used for this

Reply:
But also, shouldn't it be higher than 2MiB?

Reply:
Our decision was to allow setting any size; 2MB is just the max for common schema logs.  There is an undocumented exception that occurs when you set it to 16384 or lower, though, so it's to prevent that exception from occurring.  I agree that since you can't in practice use 16385-20479 byte sizes then it just makes sense to use 20480.  I'll update the value and the comment to be more explicit.